code/style.css

```diff
@@ -2,6 +2,308 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
+/*.active + .answer {
   /* some styling here */
+/*} */
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Poppins&family=Rubik:wght@800&display=swap');
+
+*, *::before, *::after {
+    box-sizing: border-box;
+    outline: none; /*remove the outline when click on the form input boxes */
+  }
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  background-color: #131212;
+  color: rgba(245, 245, 245, 0.863);
+  width: 100%;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  margin: 20px 0;
+}
+  
+/************** 1. HERO SECTION **************/
+  .hero-img{
+    position: relative; /* PARENT NEEDS TO BE RELATIVE */
+    background-color: #131212;
+}
+  
+.hero-img img {
+  width: 100vw;
+  padding: 0;
+  height: 50vh;
+  background-position: center;
+  object-fit: cover;
+  filter: brightness(90%); /* make the picture a bit darker for the text to pop */
+}
+  
+.hero-text{
+  position: absolute;
+  top: 0; /* all on zero to make the div cover whole image*/
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 0 5px;
+  /***** styling ****/
+  color: rgba(255, 255, 255, 0.904);
+  font-family:'Rubik', sans-serif;
+}
+
+.hero-text h1 {
+/***** styling ****/
+  font-size: 2rem;
+  font-weight: 900;
+  white-space: nowrap;
+  padding: 10px;
+  margin: 0 5px;
+}
+
+.hero-text h2 {
+  font-size: 1.2rem;
+  margin: 0;
+}
+
+/************** 2. SIGN UP FORM **************/
+
+.form-wrapper {
+  width: 100vw;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #131212;
+}
+
+.contact-form {
+  width: 80vw;
+  margin: 0px;
+  display: flex;
+  padding: 5px 20px;
+  /***** styling ****/
+  background-color: rgba(29, 26, 27, 0.575);
+  border-radius: 5px;
+  box-shadow: 0 0 5px 5px rgba(255, 6, 110, 0.445);
+  font-size: 14px;
+  color: rgba(255, 255, 255, 0.774);
+}
+
+.input-fields { 
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+}
+
+.input-fields ul { /* No bullets list */
+  list-style-type: none;
+  padding: 0;
+}
+
+.input-fields li {
+  margin-bottom: 10px;
+}
+
+.input-fields, .input-box {
+  width: 100%;
+  margin: 10px;
+  padding: 5px;
+  background: transparent;
+  border: 0;
+  border-bottom: 2px solid rgba(255, 6, 102, 0.575);
+  color: rgba(255, 255, 255, 0.774);
+}
+
+.input-news {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.input-news, .news-checkbox {
+  margin-right: 10px;
+}
```

Comment:
Not sure this is what you meant, but you should be able to make this bigger just by setting a width and height to it! But I agree, it's kind of tricky to style forms in general but you did a great job!